java/src/main/java/org/rocksdb/UInt64AddOperator.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2018, Vlad Balan (vlad.gm@gmail.com).  All rights reserved.
```

Comment:
Please remove this copyright line or change it to:
```
Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Reply:
will do - I just copied the header from StringAppendOperator.java - would you like me to change that one too?

Reply:
Nope, leave the StringAppendOperator.java one as it is.

Reply:
done.